### PR TITLE
Missing include guards

### PR DESCRIPTION
**Pull request by rbuj**
Reported by Apache NetBeans IDE:

### Before
Captura de Pantalla 2021-02-24 a les 12 29 12

### After
Captura de Pantalla 2021-02-24 a les 12 32 10

### CODE REVIEW

File: capplets/display/scrollarea.h [hunk 1]
```diff
@@ -18,6 +18,9 @@
 #include <cairo.h>
 #include <gtk/gtk.h>
 
+#ifndef __SCROLLAREA_H__
```
Comment (zhuyaliang) [inline capplets/display/scrollarea.h:21]: Move to the first line at the beginning of the code

Comment (rbuj) [reply capplets/display/scrollarea.h:19]: done

File: capplets/about-me/mate-about-me-fingerprint.h [hunk 2]
```diff
@@ -19,9 +19,13 @@
 
 #include <gtk/gtk.h>
 
+#ifndef __MATE_ABOUT_ME_FINGERPRINT_H__
```
Comment (zhuyaliang) [inline capplets/about-me/mate-about-me-fingerprint.h:22]: I think it should also be on the first line of the code

Comment (rbuj) [reply capplets/about-me/mate-about-me-fingerprint.h:20]: done

**Review by zhuyaliang — APPROVED** [review]

**Pull request merged**